src/components/MobileAccessGuard/MobileGuardWrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box, useMediaQuery } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import { PropsWithChildren } from 'react'
+import { MobileGuardWarning } from './MobileGuardWarning'
+
+// restrict access on below md resolution
+export const MobileGuardWrapper = ({ children }: PropsWithChildren) => {
+  const theme = useTheme()
+  const notDesktop = useMediaQuery(theme.breakpoints.down('md'))
+
+  if (notDesktop) {
+    return <MobileGuardWarning />
+  }
+
+  return <Box sx={{ display: { xs: 'none', md: 'block' } }}>{children}</Box>
```

Comment:
```suggestion
  return <Box sx={{ display: { xs: 'none', md: 'block' } }}>{children}</Box>
```
udah ga perlu lg display disini karena kan udah conditional rendering